### ISSUE TITLE
V3 monitoring pipeline's MonitoringProcessor is slow

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#5267]
We had a lot of activity on V3 this morning and the `MonitoringProcessor` has fallen behind (further than any of our other jobs). I believe this is due to some issue with making many many duplicate web requests and waiting on them individually, but it's not clear. There need to be some performance improvements made to the job so it can keep up.

**Comment by skofman1** [comment]
This is a duplicate of: https://github.com/NuGet/NuGetGallery/issues/6327
